Tasks/Gradle/CodeAnalysis/gradlesonar.ts

```diff
@@ -58,14 +53,19 @@ export function applyEnabledSonarQubeArguments(gradleRun: trm.ToolRunner):trm.To
 
 // Points SonarQube to the CC file as it is in a non-standard location. Not required for Jacoco. 
 export function applySonarQubeCodeCoverageArguments(gradleRun: trm.ToolRunner, isCodeCoverageEnabled:boolean, ccTool:string, reportPath:string):trm.ToolRunner {
-    
-    if (isSonarQubeEnabled() && isCodeCoverageEnabled && ccTool.toLowerCase() == "cobertura" && reportPath ) {
+    if (!sqCommon.isSonarQubeAnalysisEnabled()) {
```

Comment:
This wasn't meant to be in the commit. However, this brings up the idea that perhaps we should move all the boolean checks up here in the return-early pattern.